.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Proposed Changes
+
+_(Describe the changes provided by the pull request here.)_
+
+## Issues
+
+_(If applicable, add links to any issues resolved.)_
+
+## Types of changes
+
+_What types of changes does the pull request introduce? Put an `x` in the boxes that apply:_
+
+- [ ] Bug/issue fix (non-breaking change that fixes an issue).
+- [ ] New feature added (non-breaking change that adds functionality).
+- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
+- [ ] Documentation update.
+- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).
+
+## Checklist
+
+_Put an `x` in the boxes that apply:_
+
+- [ ] Checks and tests pass locally.
+- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) resource.
```

Comment:
reorder the above 2

Reply:
It is not clear what you mean here. Have changed order of 2 options given